pep-0690.rst

```diff
@@ -131,9 +131,26 @@ dictionaries in general.
 Specification
 =============
 
-Lazy imports are opt-in, and globally enabled either via a new ``-L`` flag to
-the Python interpreter or via a call to a new ``importlib.set_lazy_imports()``
-function, which makes all imports after the call lazy.
+Lazy imports are opt-in, and they can be:
+
+* Globally enabled, either via a new ``-L`` flag to the Python interpreter or
+  via a call to a new ``importlib.set_lazy_imports()`` function, which makes
+  *all* new imports after the call immediately lazy.
```

Comment:
this is not accurate. "*all*" implies, well, "all" :) but in practice it applies to the same imports that would have been lazy under `-L` (e.g. module-level etc)